addons/website_sale/static/src/js/cart.js

```diff
@@ -100,4 +101,38 @@ publicWidget.registry.websiteSaleCart = publicWidget.Widget.extend({
     },
 });
 
-export default publicWidget.registry.websiteSaleCart;
+publicWidget.registry.websiteSaleCartNavigation = publicWidget.Widget.extend({
+    selector: '.o_website_sale_checkout',
+
+    /**
+     * For mobile screens, `.o_cta_navigation_container` has an absolute position causing
+     * overlapping issues with nearby divs, therefore the height of `.o_website_sale_checkout` needs
+     * to include the height of the absolute div and needs to be updated every time an element on
+     * the checkout is expanded (i.e. payment methods, cart summary)
+     *
+     * @override
+     */
+    start() {
+        const ctaNavigation = document.querySelector('.o_cta_navigation_container')
+        if (uiUtils.isSmall() && ctaNavigation) {
+            const updateCheckoutHeight = () => {
+                const updatedHeight = ctaNavigation.offsetTop + ctaNavigation.offsetHeight
+                this.el.style.height = `${updatedHeight}px`;
+            }
+            this.resizeObserver = new ResizeObserver(updateCheckoutHeight);
+            const paymentForm = document.getElementById('o_payment_form')
+            const cartSummary = document.getElementById('o_wsale_accordion_item')
+            if (paymentForm){
+                this.resizeObserver.observe(paymentForm);
+            }
+            if (cartSummary){
+                this.resizeObserver.observe(cartSummary);
+            }
+        }
+    },
```

Comment:
You can cause a memory leak if you don't destroy the observer because it will be instantiated on every page, leading to multiple observers staying open in the browser each time a customer goes through checkout in an Odoo eCommerce.
```suggestion
    },
    destroy() {
        super.destroy();
        this.resizeObserver.disconnect();
    }
```